tools/libxl/libxl_arm.c

```diff
@@ -29,6 +29,7 @@ int libxl__arch_domain_prepare_config(libxl__gc *gc,
     unsigned int i;
     uint32_t vuart_irq;
     bool vuart_enabled = false;
+    libxl_device_vgsx *vgsx;
```

Comment:
if you want this helper variable then put it down into
`if (d_config->num_vgsxs) {`
where it is only used

Reply:
ok